### PR TITLE
Antd v4 to v5 upgrade

### PR DESCRIPTION
**Pull request by samouzegar10**

### CODE REVIEW

File: frontend/src/components/datasets/ExpandableTableDatasetComments.tsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ import { Dataset } from "../../models/frontend_models";
 export default function ExpandableTableDatasetComments(): NonNullable<TableProps<any>['expandable']> {
   return (
     {
-      columnTitle: <div>Comments</div>,
+      columnTitle: <CommentOutlined />,
```
Comment (UlysseFG) [inline frontend/src/components/datasets/ExpandableTableDatasetComments.tsx:12]: Maybe add a hovering tooltip "Validation comments" to the icon ?

Comment (samouzegar10) [reply frontend/src/components/datasets/ExpandableTableDatasetComments.tsx]: tooltip added

File: frontend/src/components/studySamples/CompletedSamplesTable.tsx [hunk 2]
```diff
@@ -103,6 +103,7 @@ function CompletedSamplesTable({studyID, step, workflowAction, tableState, setti
 				USER,
 				COMMENT
 			]}
+      scroll={{ x: 1500, y: 700}}
 			dataSource={dataSource}
```
Comment (nafiz1001) [inline frontend/src/components/studySamples/CompletedSamplesTable.tsx:107]: indentation

**Review by nafiz1001 — APPROVED** [review]

**Review by UlysseFG — APPROVED** [review]
There is some fixes to be done in another PR but let's merge that one...

**Pull request merged**